tests/test_views.py

```diff
@@ -9,13 +9,19 @@ def setUp(self):
         self.client = Client()
         self.key = "key"
         self.key2 = "key2"
+        self.key3 = "key3"
         self.value = "value"
         self.value2 = "value2"
-        self.option = Option.objects.create(key=self.key, value=self.value)
-        self.option2 = Option.objects.create(key=self.key2, value=self.value2)
+        self.value3 = "value3"
+        self.public = False
+        self.public2 = True
+        self.public3 = True
```

Comment:
Since we don't need to refer to the public status of an individual option, you should be able to delete these lines and just set the booleans directly in the `Option.objects.create(xyz)` lines